dev/src/transaction.ts

```diff
@@ -117,7 +118,7 @@ export class Transaction {
 
     if (refOrQuery instanceof DocumentReference) {
       return this._firestore
-          .getAll_([refOrQuery], this._requestTag, this._transactionId)
+          .getAll_([refOrQuery], null, this._requestTag, this._transactionId)
```

Comment:
Can you label this `null`? In this context it's a lot like a `boolean` with no clear meaning.

Or add something like `const fieldMask = null` and then pass that?